docs/main/user-guides/replicated-storage-user-guide/additional-information/i-o-path-description.md

```diff
@@ -12,15 +12,15 @@ description: This section provides an overview of the topology and function of t
 
 ### Replicated Storage Instance
 
-An instance of the `mayastor` binary running inside a Replicated Storage (a.k.a Replicated Engine and f.k.a Mayastor) container, which is encapsulated by a Replicated Storage pod. 
+An instance of the `mayastor` binary running inside an IO engine pod, which is encapsulated by an IO engine pod.
```

Comment:
first is container, then is pod.
"inside an IO engine container, which is encapsulated by an IO engine pod."